src/test/kotlin/org/rust/lang/core/resolve/RsStubOnlyResolveTest.kt

```diff
@@ -561,4 +561,24 @@ class RsStubOnlyResolveTest : RsResolveTestBase() {
     //- main/foo.rs
         pub struct Bar;
     """, NameResolutionTestmarks.crateRootModule)
+
+    fun `test scoped resolve inside stubbed function body`() = stubOnlyResolve("""
```

Comment:
What does this test case check?
Looks like it already works with the current nightly plugin

Reply:
I will not work if `foo.rs` is stubbed

Reply:
Indeed!